homeassistant/components/lg_thinq/const.py

```diff
@@ -10,3 +11,10 @@
 THINQ_PAT_URL: Final = "https://connect-pat.lgthinq.com"
 CLIENT_PREFIX: Final = "home-assistant"
 CONF_CONNECT_CLIENT_ID: Final = "connect_client_id"
+
+# MQTT
+MQTT_SUBSCRIPTION_INTERVAL: Final = timedelta(days=180)
```

Comment:
Why do we set it to this high value? I don't expect a majority running their HA for 180 days straight

Reply:
Changed to 1.